kiwi/bootloader/config/__init__.py

```diff
@@ -34,22 +36,35 @@ class BootLoaderConfig:
     :param string root_dir: root directory path name
     :param dict custom_args: custom bootloader config arguments dictionary
     """
-    def __new__(
-        self, name, xml_state, root_dir, boot_dir=None, custom_args=None
+    @abstractmethod
+    def __init__(self) -> None:
+        return None                     # pragma: no cover
```

Comment:
This is nice, didn't know that way of skipping the code coverage. I have added it to the kiwi/test/unit/.coveragerc under report/omit. But your solution is better :+1:

Reply:
Yes I did some trials to avoid it having to omit any line, but I had no success :P I was also not aware of it way of omitting codes I discovered it yesterday trying to figure out some way to handle the abstract methods coverage.